Yubico.YubiKey/src/Yubico/YubiKey/Cryptography/AesUtilities.cs

```diff
@@ -139,6 +136,8 @@ public static byte[] AesCbcEncrypt(byte[] key, byte[] iv, ReadOnlySpan<byte> pla
                 }
             }
 
+            CryptographicOperations.ZeroMemory(aesObjKey);
+            CryptographicOperations.ZeroMemory(aesObjIv);
```

Comment:
If an exception is thrown after aesObjKey and aesObjIv are assigned (lines 114, 115), then these two lines will not be called. To make sure you always zero out the memory, use a try/finally block. Exceptions can be thrown from `CryptoStream.Write(byte[] buffer, int offset, int count)`.